server/models/account/account-video-rate.ts

```diff
@@ -185,4 +196,11 @@ export class AccountVideoRateModel extends Model<AccountVideoRateModel> {
       else if (type === 'dislike') await VideoModel.increment({ dislikes: -deleted }, options)
     })
   }
+
+  toFormattedJSON (): UserVideoRate {
+    return {
+      videoId: this.videoId,
```

Comment:
`this.Video.toFormattedJSON()` (you'll have to create an `AccountVideoRate` type)